graphql/resolvers.go

```diff
@@ -2785,13 +2785,17 @@ func (r *queryResolver) MainlineCommits(ctx context.Context, options MainlineCom
 type versionResolver struct{ *Resolver }
 
 // Returns task status counts (a mapping between status and the number of tasks with that status) for a version.
-func (r *versionResolver) TaskStatusCounts(ctx context.Context, v *restModel.APIVersion) ([]*StatusCount, error) {
+func (r *versionResolver) TaskStatusCounts(ctx context.Context, v *restModel.APIVersion, options *BuildVariantOptions) ([]*StatusCount, error) {
 	defaultSort := []task.TasksSortOrder{
 		{Key: task.DisplayNameKey, Order: 1},
 	}
 	opts := data.TaskFilterOptions{
-		Sorts: defaultSort,
+		Statuses:  options.Statuses,
+		Variants:  options.Variants,
+		TaskNames: options.Tasks,
+		Sorts:     defaultSort,
```

Comment:
This also accepts a `FieldsToProject` option where you can pass the DisplayStatus Field so it will only return the displayStatus instead of including all of the task fields since we only care about the displayStatus